conductor/submitter.py

```diff
@@ -1877,12 +1878,6 @@ def group_contiguous_frames(cls, frames, ends_only=False):
         return ranges
 
 
-class UserCanceled(Exception):
```

Comment:
Just wondering what the thought was behind moving this exception to from `submitter.py` to  `common.py`. I guess it _is_  so generically named/constructed that it could be re/used in a variety of contexts and still make sense.  But I don't think you're using it outside of the submitter, yah?  (`submitter_maya.py` inherits/imports from `submitter.py`,  so i think it already has access to the  `UserCanceled` exception without moving it, yah?) .  Not a big deal either way :)

Reply:
I'd like to suggest we move the exception in an `exceptions` module, eg. `conductor.lib.exceptions`, and suffixed with `Error`, like the built-in exceptions, eg. `UserCanceledError`.

Reply:
Good idea - common felt like a 'meh' place to put it.  I've created an exceptions module and put the UserCanceledError exception there.   There are more custom exceptions we should move into that module - but that is outside the scope of this PR.   I'll put together another JIRA ticket to address that refactor.

Reply:
@flebel you read my mind